site/_layouts/docs.html

```diff
@@ -50,14 +50,18 @@ <h3>{{ group.title }}</h3>
 </div>
 
 <script src="{{site.baseurl}}{{page.demos_script_link}}/dist/index.js?v={{ site.time | date: '%s' }}"></script>
-<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"></script>
 {% assign product_link_parts = page.product_link | split: "/" %}
-<script type="text/javascript">
-  docsearch({
+<script
+  type="text/javascript"
+  src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"
+  onload="docsearch({
```

Comment:
`onload` accepts a function. Does `docsearch(...)` returns a function? Or should it be something like this?

```html
onload="function() { docsearch(...); }"
```